async-voter.js

```diff
@@ -72,25 +73,45 @@ module.exports = (app, repository) => {
     const actions = payload.actions
     const text = payload.original_message.text
     const user = payload.user.name
+    const user_id = payload.user.id
     const channel_id = payload.channel.id
 
-    repository.get(channel_id, (err, reply) => {
+    const extractTicketDescription = (text) => {
+      return text.replace(/^<!here> ASYNC VOTE on \"/,'').replace(/"$/,'')
+    }
+
+    const ticket_description = extractTicketDescription(text)
+
+    repository.get(channel_id + "-" + ticket_description, (err, reply) => {
+
       const votes = JSON.parse(reply) || {}
 
       if (actions[0].value === 'reveal') {
-        res.send(formatResult(text, votes))
+        repository.set(channel_id+"-"+ticket_description+"-revealed", JSON.stringify({'user-voting-session-revealor' : user_id, 'timestamp-vote-revealed': new Date().toISOString()}), (err, reply) => {
```

Comment:
would another approach be to have everything stored under a single key, e.g. `channel_id-ticket_description` and then store all the `revealed` and other user votes in the JSON object that's stored under that key?

At least that would mean if keys are pruned from the cache we don't lose parts of a vote - i.e. everything related to a particular vote is stored under one key ...?